cmd/baseserver/README.md

```diff
@@ -24,11 +24,17 @@ Route | Method | Completed | Description
 /sign|POST|✔️|Sign a transaction
 /tx|POST|✖️|Post a transaction to the blockchain
 /seeds/status|GET|✖️|Returns the information on the last seed
+/build/create_role|POST|✔️|Creates a role. Please note that the role MUST be valid hex for example instead of sending "role", send its hex encoded equivalent "726f6c65"
```

Comment:
Yeah, we use hex identifiers everywhere and there was confusion writing tests with it not being hex.  Then in a discussion, someone mentioned it was not good to use strings ("cosmos-devs") but rather force hex bytes which would generally be chosen with no human significance, and not leak information.  

Seems easier and no need to use human-readable strings in the API, although one could build that on top, if so desired.